README.md

```diff
@@ -1,13 +1,90 @@
+- Posters can also be stored for later viewing by clicking `Save This Poster`. This switches to a grid view displaying mini versions of all saved posters.
+![saved posters view + delete gif](link in the parens)
+- To delete a poster, the user can double click the image.
+- This applications runs directly in the web browser!
+
+#### Want to contribute?
+- Interested in contributing to Hang in there?!
+- Fork this project and clone to your machine.
+- Create a pull request, and one of our project managers will get back to you with a code review!
+
+#### Technologies Used
+- JavaScript
+- HTML
+- CSS
+
+#### Project Reflection
+##### Nichele
+- I had a great experience during this project. Peter and I spent a good deal of time planning and discussing the problems, challenges we might face, and how to address problems when encountered.
+##### Peter
+
+##### Wins
+##### Nichele
+- I felt like our communication styles worked well together, and we were able to address coding challenges through discussion and solo pseudocoding.
+- Planning! We spent a lot of time in discussion deciding how to address a problem before writing any code. This seemed to give us a clear direction without leaving us muddled in issues we hadn't accounted for.
+- Our git workflow seemed to come together pretty quickly and I didn't have any issues with merges or pulls.
+- We were able to reach out to mentors when we needed assistance, and were able to come up with a solution to our problem with some gentle nudging.
+##### Peter
+
+##### Challenges
+##### Nichele
+- I wish I had taken more notes! On the next project, I think I'll create a separate file within the project to store reflections and questions that come up during the project.
+- We didn't really utilize our mentors for code reviews. Though we did discuss issues we were having, we didn't actually add them to any of the PR code reviews. Next time I would like to discuss a code review schedule with my mentors prior to beginning the project.
+##### Peter
+
+##### Overall Impressions
+##### Nichele
+- Overall this project went well for me! I was able to gain more experience working with another developer, and continue to look at what my interactions are like in those situations.
+- I feel good about my progress in understanding the JavaScript logic and why certain solutions might be better than others.
+- I was able to understand the DOM and how it interacted with our code, and how to add and remove elements from the page after the initial DOM tree was created.
+- I learned a lot about what a good design and planning phase could mean for the future of a project and my own sanity during development! 
+##### Peter
+
```

Comment:
- This was an ideal first "Paired Project" and I feel fortunate that I had a partner who  communicated well with me, from beginning to end.  I learned a great deal about collaborative coding and the overall experience of working on a team, and I'm grateful that I was able to do so not through conflict or frustration, but through recognition of all the things that actually went well.
- Among the many things I learned more about and gained more experience with, the DOM (on both a conceptual and practical/interactive level) and event handlers/methods were probably the most significant.  This was also an exercise in the importance of effective "Googling", as we were able to discover and implement several JavaScript tools previously unfamiliar to us.

Reply:
√